src/SessionSetup/ServiceDeployManager.cpp

```diff
@@ -385,6 +390,16 @@ void ServiceDeployManager::CopyFileToLocalImpl(orbit_base::Promise<ErrorMessageO
       return;
     }
 
+    if (stop_token->IsStopRequested()) {
+      ErrorMessageOr<bool> remove_result = orbit_base::RemoveFile(destination);
+      if (remove_result.has_error()) {
+        ORBIT_ERROR("Error while deleting partially downloaded file: %s",
+                    remove_result.error().message());
+      } else if (!remove_result.value()) {
+        ORBIT_ERROR("Unable to delete partially downloaded file");
+      }
+    }
```

Comment:
This could also be done somewhere else.. for example in `SftpCopyToLocalOperation` or in OrbitMainWindow or OrbitApp. Whats the best location?

Reply:
Maybe `SftpCopyToLocalOperation`. Would make sense to me to handle the deletion of a partially copied file there. Not sure what other implications that might have.